sql/core/src/main/protobuf/org/apache/spark/status/protobuf/store_types.proto

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+package org.apache.spark.status.protobuf.sql;
```

Comment:
a newbie question: Can we import the `store_types.proto` file defined in the core module here? Otherwise, we need to repeatedly define some data structures in the future , such as `enum JobExecutionStatus`.

Reply:
seems impossible

Reply:
Ya I don't think that will work.
We can always reference the java files it creates so should get access to all the structures.